vscode/package.json

```diff
@@ -473,7 +473,29 @@
       "[qsharp]": {
         "debug.saveBeforeStart": "none"
       }
-    }
+    },
+    "extensionDependencies": [
+      "github.copilot-chat"
```

Comment:
Being this is a new dependency, do you know what the behavior is if we ship this in an update and the user doesn't already have (or can't install for some reason) this extension? Do we fail to install? Can it be optional if so?

Reply:
I've tested it on VS Code where the github copilot is disabled, and the extension works fine. It just doesn't provide the quantum copilot experience (because that part needs the copilot extension).